### ISSUE TITLE
Add support for "--consistency" option

### ISSUE THREAD

**Issue by janisto** [ert78gb/google-datastore-emulator#15]
More info:
  --consistency=CONSISTENCY
    The fraction of job application attempts that will succeed, with 0.0
    resulting in no attempts succeeding, and 1.0 resulting in all attempts
    succeeding. Defaults to 0.9. Note that setting this to 1.0 may mask
    incorrect assumptions about the consistency of non-ancestor queries;
    non-ancestor queries are eventually consistent.

**Comment by ert78gb** [comment]
I could add it on tomorrow afternoon or Wednesday.

**Comment by ert78gb** [comment]
Could you write a good test case? I understand the problem and I added the new option (not committed yet), but I would like to write unit test to 0.0, 0.9 and 1.0 behaviour. Maybe you have exists unit test that I could include.

**Comment by janisto** [comment]
I think 0.9 would be hard to test. I'm using Jest with my unit tests, so it would look something like this:

```js
'use strict';

/* global jest, jasmine, describe, it, beforeAll, afterAll, beforeEach, afterEach, expect */

const Datastore = require('@google-cloud/datastore');
const Emulator = require('google-datastore-emulator');
const path = require('path');

// Increase timeout
jasmine.DEFAULT_TIMEOUT_INTERVAL = 20000;

describe('Test Datastore', () => {
  // Each test suite needs to have a unique folder.
  const emulatorDir = path.resolve('./test/ds-test');
  let emulator;
  let ds;

  beforeAll(() => {
    const options = {
      // consistency: 1.0,
      port: 9090, // Each test suite needs to have a unique port.
      dataDir: emulatorDir
    };
    emulator = new Emulator(options);
    return emulator.start()
    .then(() => {
      // console.log('Emulator ready. Host: ', process.env.DATASTORE_EMULATOR_HOST);
      ds = Datastore({
        keyFilename: {},
        projectId: 'test'
      });
      return true;
    });
  });

  afterAll(() => {
    return emulator.stop();
  });

  describe('Test save', () => {
    it('Save promise should reject', () => {
      const key = ds.key(['TestData1']);
      const data = {
        title: 'Test1'
      };

      return expect(ds.save({
        key: key,
        data: data
      })).rejects.toBeDefined();
    });

    it('Save promise should resolve', () => {
      const key = ds.key(['TestData2']);
      const data = {
        title: 'Test2'
      };

      return expect(ds.save({
        key: key,
        data: data
      })).resolves.toBeDefined();
    });
  });
});
```

**Comment by ert78gb** [comment]
I published 1.2.0 version that contains this options.

I can not reproduce reject test case
